test/unit/gateways/braintree_blue_test.rb

```diff
@@ -921,7 +921,7 @@ def test_that_setting_a_wiredump_device_on_the_gateway_sets_the_braintree_logger
     end
   end
 
-  def test_solution_id_is_added_to_create_transaction_parameters
+  def test_channel_is_added_to_create_transaction_parameters
```

Comment:
how come this is renamed?

Reply:
`solution_id` was the old name for the field at BT. It's now called `channel` so for future people reading tests this will make more sense